packages/create-project/package.json

```diff
@@ -39,6 +40,7 @@
     "@types/prompts": "2.0.14",
     "@types/rimraf": "^3.0.2",
     "@types/tar": "^6.1.1",
+    "@types/uuid": "^8.3.4",
```

Comment:
Does uuid still require `types`?

Reply:
tried removing 

<img width="994" alt="image" src="https://user-images.githubusercontent.com/251288/194669777-265c1242-d301-4395-b656-bd184efde44c.png">

Reply:
Just use crypto randomUUID in tests